pages/linux/trizen.md

```diff
@@ -0,0 +1,27 @@
+# trizen
+
+> Arch Linux utility for building packages from the Arch User Repository (AUR).
+
+- Synchronize and update all AUR packages (excludes non-AUR packages):
+
+`trizen -Syua`
+
+- Install a new package (includes AUR packages):
+
+`trizen -S {{package}}`
+
+- Remove a package and its dependencies (includes AUR packages):
+
+`trizen -Rs {{package}}`
+
+- Search the package database for a keyword (including AUR):
+
+`trizen -Ss {{package}}`
```

Comment:
Should it be "keyword" instead of "package" ? Because you are entering the keyword to search for the package.